pinot-core/src/main/java/org/apache/pinot/core/data/manager/offline/TableDataManagerProvider.java

```diff
@@ -51,13 +51,21 @@ public static TableDataManager getTableDataManager(@Nonnull TableDataManagerConf
       @Nonnull String instanceId, @Nonnull ZkHelixPropertyStore<ZNRecord> propertyStore,
       @Nonnull ServerMetrics serverMetrics, @Nonnull HelixManager helixManager) {
     TableDataManager tableDataManager;
-    switch (TableType.valueOf(tableDataManagerConfig.getTableDataManagerType())) {
+    switch (TableDataManagerType.valueOf(tableDataManagerConfig.getTableDataManagerType())) {
       case OFFLINE:
+        // TODO remove: temporary hack for testing
+        if (tableDataManagerConfig.getTableName().equals("dimBaseballTeams_OFFLINE")) {
```

Comment:
remove this?

Reply:
Will be removed when rebasing from Dharak's diff.

Reply:
Done.